src/editor/canvas.js

```diff
@@ -127,6 +127,15 @@ export function scaleCanvas(canvas, size, scale = 1) {
   canvas.width = size * scale;
   canvas.height = size * scale;
   const ctx = canvas.getContext('2d');
+  try {
+    // Try to improve the image quality when shrinking large images.
+    if ('imageSmoothingEnabled' in ctx)
```

Comment:
I think the if statements can be removed. If the properties aren't supported, they should just do nothing.